gensim/models/atmodel.py

```diff
@@ -68,6 +68,18 @@ class AuthorTopicState(LdaState):
     """
```

Comment:
missing docstring for module (examples of usage, liks to related papers, etc)

Reply:
I have added a usage example in the docstring.

Reply:
what's about related papers/links/etc? add one more example - almost always a good idea

Reply:
I have added the paper on Author topic model to the top of the script.